src/proposals/api/views.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import views
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+from core.authentication import TokenAuthentication
+from proposals.models import TalkProposal, TutorialProposal
+from reviews.models import Review
+
+
+class ProposalAPIView(views.APIView):
+
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        num_proposal_talk = TalkProposal.objects.filter(cancelled=False).count()
+        num_proposed_tutorial = TutorialProposal.objects.filter(cancelled=False).count()
+        num_stage_1_reviews = Review.objects.filter(stage=1).count()
+        num_stage_2_reviews = Review.objects.filter(stage=2).count()
+
+        response_data = {}
+        response_data[num_proposal_talk] = num_proposal_talk
+        response_data[num_proposed_tutorial] = num_proposed_tutorial
+        response_data[num_stage_1_reviews] = num_stage_1_reviews
+        response_data[num_stage_2_reviews] = num_stage_2_reviews
```

Comment:
是不是該寫個 unit test XD

```python
num_proposal_talk = 1
response_data = {}
response_data[num_proposal_talk] = num_proposal_talk
print(response_data)  # {1: 1}
```

Reply:
建議直接改成一個 statement 就好
```suggestion
        response_data = {
            "num_proposal_talk": num_proposal_talk,
            ...
        }
```

Reply:
fixed ~

Reply:
added unit test